tests/ts_simple_describe_table.erl

```diff
@@ -38,9 +38,9 @@ confirm() ->
     Expected =
         {[<<"Column">>,<<"Type">>,<<"Is Null">>,<<"Primary Key">>, <<"Local Key">>],
         [{<<"myfamily">>,  <<"varchar">>,   false,  1,  1},
-        {<<"myseries">>,   <<"varchar">>,   false,  2,  2},
-        {<<"time">>,       <<"timestamp">>, false,  3,  3},
-        {<<"weather">>,    <<"varchar">>,   false, [], []},
-        {<<"temperature">>,<<"double">>,    true,  [], []}]},
+         {<<"myseries">>,   <<"varchar">>,   false,  2,  2},
+         {<<"time">>,       <<"timestamp">>, false,  3,  3},
+         {<<"weather">>,    <<"varchar">>,   false, undefined, undefined},
+         {<<"temperature">>,<<"double">>,    true,  undefined, undefined}]},
```

Comment:
I think so. In the original pb terms, `undefined`s are converted to `[]` in riak_pb_ts_codec, iirc.